core/cmd/lambda/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/kislerdm/diagramastext/core"
+)
+
+type Request struct {
+	Prompt string `json:"prompt"`
+}
+
+func main() {
+	clientOpenAI, err := core.NewOpenAIClient(
+		core.ConfigOpenAI{
+			Token:       os.Getenv("OPENAI_API_KEY"),
```

Comment:
**question** what do you think about moving these env variable names to an `openapi` internal package and holding these as constants there? Otherwise, we might them spread all around, which will be hard to maintain.

Reply:
That would be against clear architecture with no clear benefit. Function initializing the client shall not hide critical options - package user shall be able to configure them explicitly.